packages/panna-sdk/src/react/components/buy/select-buy-provider-step.tsx

```diff
@@ -45,7 +45,7 @@ export function SelectBuyProviderStep({ form }: SelectBuyProviderStepProps) {
     error: quoteError
   } = useOnrampQuotes({
     tokenSymbol: token?.symbol || '',
-    network: currentChain.name?.toLowerCase() || 'lisk',
+    network: ONRAMP_NETWORK,
     fiatAmount: fiatAmount || 0,
     fiatCurrency: DEFAULT_CURRENCY
```

Comment:
- `network` should be derived from the current chain. Please make use of the current chainId, determine the name of the chain accordingly and then apply `toLowerCase()`.
- `fiatCurrency` should come from user inputs and not hardcoded to the default.